charts/pega/charts/constellation/templates/clln-deployment.yaml

```diff
@@ -18,7 +18,13 @@ spec:
         app: constellation
     spec:
       imagePullSecrets:
+{{- if .Values.global.docker.image_pull_secret_names }}
```

Comment:
Can you create template for below few lines of code re-use at other places?

Reply:
Created template and referred it in deployments and statefulsets.